### DISCUSSION TITLE
Can I provide a parameter to PhysicalFileProvider

### DISCUSSION THREAD

**Discussion by decie** [dotnet/aspnetcore#29085] [Q&A]
I want to show some files, these files are in ftp server. Can I pass a ftp address to PhysicalFileProvider, then it can access the files like  the local directory.
FileProvider = new PhysicalFileProvider(@"\\*.*.*.*\dir"),

**Comment by Tratcher** [comment]
PhysicalFileProvider uses the System.IO.File APIs and I don't think those support FTP access. You'd need to write a new FtpFileProvider.
